packages/next-app/src/app/stableswapsimulator/page.tsx

```diff
@@ -1,7 +1,15 @@
 "use client";
 
+import { useEffect } from "react";
+
+import { useStableSwap } from "#/contexts/StableSwapContext";
+
 import ParametersNotSet from "./(components)/ParametersNotSet";
 
 export default function Page() {
+  const { setInitialData, defaultInitialData } = useStableSwap();
+  useEffect(() => {
+    setInitialData(defaultInitialData);
+  }, []);
```

Comment:
Shouldn't this be called from within `useStableSwap` or shouldn't we have the default initial value as this?